.github/CODEOWNERS

```diff
@@ -0,0 +1,428 @@
+/llvm/tools/llvm-debuginfod-find @llvm/pr-subscribers-debug-info
+/llvm/lib/CodeGen/AsmPrinter @llvm/pr-subscribers-debug-info
+/clang/lib/CodeGen/CGDebugInfo.cpp @llvm/pr-subscribers-debug-info
+/llvm/include/llvm/BinaryFormat/Dwarf.* @llvm/pr-subscribers-debug-info
+/llvm/test/DebugInfo @llvm/pr-subscribers-debug-info
+/llvm/test/tools/dsymutil @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-debuginfo-analyzer @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-debuginfod @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-debuginfod-find @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-dwarfdump @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-dwarfutil @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-dwp @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-gsymutil @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-pdbuti @llvm/pr-subscribers-debug-info
+/llvm/lib/IR/Debug*.cpp @llvm/pr-subscribers-debug-info
+/llvm/include/llvm/IR/Debug*.h @llvm/pr-subscribers-debug-info
+
+# github-workflow
+/.github/workflows/ @llvm/pr-subscribers-github-workflow
+
+# flang-driver
+/flang/tools/flang-driver @llvm/pr-subscribers-flang-driver
+/flang/unittests/Frontend @llvm/pr-subscribers-flang-driver
+/flang/lib/FrontendTool @llvm/pr-subscribers-flang-driver
+/flang/lib/Frontend @llvm/pr-subscribers-flang-driver
+/flang/include/flang/Frontend @llvm/pr-subscribers-flang-driver
+/flang/include/flang/FrontendTool @llvm/pr-subscribers-flang-driver
+/flang/test/Driver @llvm/pr-subscribers-flang-driver
+
+# m68k
+/llvm/lib/Target/M68k @llvm/pr-subscribers-m68k
+/clang/lib/Basic/Targets/M68k.* @llvm/pr-subscribers-m68k
+/clang/lib/CodeGen/Targets/M68k.cpp @llvm/pr-subscribers-m68k
+
+# libcxx
+/libcxx/ @llvm/pr-subscribers-libcxx
+/runtimes/ @llvm/pr-subscribers-libcxx
+
+# libcxxabi
+/libcxxabi/ @llvm/pr-subscribers-libcxxabi
+/runtimes/ @llvm/pr-subscribers-libcxxabi
+
+# libunwind
+/libunwind/ @llvm/pr-subscribers-libunwind
+/runtimes/ @llvm/pr-subscribers-libunwind
+
+# objectyaml
+/llvm/include/llvm/ObjectYAML/ @llvm/pr-subscribers-objectyaml
+/llvm/lib/ObjectYAML/ @llvm/pr-subscribers-objectyaml
+/llvm/test/tools/obj2yaml/ @llvm/pr-subscribers-objectyaml
+/llvm/test/tools/yaml2obj/ @llvm/pr-subscribers-objectyaml
+/llvm/tools/obj2yaml/ @llvm/pr-subscribers-objectyaml
+/llvm/tools/yaml2obj/ @llvm/pr-subscribers-objectyaml
+
+# clang-analysis
+/clang/include/clang/Analysis/ @llvm/pr-subscribers-clang-analysis
+/clang/lib/Analysis/ @llvm/pr-subscribers-clang-analysis
+
+# clang-static-analyzer
+/clang/include/clang/StaticAnalyzer/ @llvm/pr-subscribers-clang-static-analyzer
+/clang/lib/StaticAnalyzer/ @llvm/pr-subscribers-clang-static-analyzer
+/clang/tools/scan-build/ @llvm/pr-subscribers-clang-static-analyzer
+/clang/utils/analyzer/ @llvm/pr-subscribers-clang-static-analyzer
+/clang/docs/analyzer/ @llvm/pr-subscribers-clang-static-analyzer
+
+# pgo
+/llvm/lib/Transforms/Instrumentation/ @llvm/pr-subscribers-pgo
```

Comment:
I suggest that we remove `/llvm/lib/Transforms/Instrumentation/` and `/llvm/test/Instrumentation/` and add the following:
```
/llvm/lib/Transforms/Instrumentation/CGProfile.cpp
/llvm/lib/Transforms/Instrumentation/ControlHeightReduction.cpp
/llvm/lib/Transforms/Instrumentation/IndirectCallPromotion.cpp
/llvm/lib/Transforms/Instrumentation/InstrProfiling.cpp
/llvm/lib/Transforms/Instrumentation/PGO*
/llvm/lib/Transforms/Instrumentation/ValueProfile*
/llvm/test/Instrumentation/InstrProfiling/
/llvm/test/Transforms/PGOProfile/
```

@snehasish